--- a/meson.build	2018-04-06 23:56:41.612785475 +0200
+++ b/meson.build	2018-04-13 11:03:31.850571571 +0200
@@ -36,7 +36,7 @@
 pixman         = dependency('pixman-1')
 libcap         = dependency('libcap')
 libinput       = dependency('libinput')
-libpam         = cc.find_library('libpam')
+libpam         = cc.find_library('pam')
 math           = cc.find_library('m')
 rt             = cc.find_library('rt')
 git = find_program('git', required: false)
